hack/kokoro-test.sh

```diff
@@ -28,6 +28,8 @@ pushd $GOPATH/src/github.com/grafeas/kritis
 echo "Check format"
 ./hack/check-fmt.sh
 
+echo "Running gcloud auth login..."
+gcloud auth login
```

Comment:
What's this for? It's not mentioned in the pull description.

Reply:
I'm trying to figure out auth in kokoro for reading from the kritis-int-test GCS bucket. It gave me an error "run gcloud auth login" so I figured I'd try it, but doesn't look like it worked :(

Will remove & try to find another way!